java/src/main/java/ai/rapids/cudf/ParquetChunkedReader.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *
+ *  Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package ai.rapids.cudf;
+
+import java.io.File;
+
+/**
+ * TODO
+ */
+public class ParquetChunkedReader implements AutoCloseable {
+  private long handle;
+
+
+  /**
+   * TODO
+   */
+  ParquetChunkedReader(long chunkSizeByteLimit, File path) {
```

Comment:
The constructors need to be public or we cannot use this outside of the ai.rapids.cudf java package.